### PR TITLE
CONTRIBUTERS.md file is added

### PR DESCRIPTION
**Pull request by Wmg20**

### CODE REVIEW

**Comment by demonkillerr** [comment]
Kindly mention the issue number.

File: CONTRIBUTERS.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- [ demonkillerr ] (https://github.com/demonkillerr)
+- [ JayeshVP24 ] (https://github.com/JayeshVP24)
+- [ Rahilsiddique] (https://github.com/Rahilsiddique)
+- [ rapid-killer-9] (https://github.com/rapid-killer-9)
+- [ riyavishwa981] (https://github.com/riyavishwa981)
+- [ RohanDarade] (https://github.com/RohanDarade)
+- [ Shreyas30] (https://github.com/Shreyas30)
+- [ ShudhanshuSingh] (https://github.com/ShudhanshuSingh)
+- [ Vansh-Baghel] (https://github.com/Vansh-Baghel)
+- [VedantPol] (https://github.com/VedantPol)
```
Comment (demonkillerr) [inline CONTRIBUTERS.md]: Kindly check the spacing. I recommend that we keep it consistent throughout.

**Review by demonkillerr — CHANGES_REQUESTED** [review]

**Comment by demonkillerr** [comment]
@kuxshl kindly mention the UI/UX team members responsible for the design.

**Comment by kuxshl** [comment]
UI/UX Team for this project:

@kuxshl 
@Mahitej28 
@SwanshTCET 
@JayeshVP24

File: CONTRIBUTERS.md (hunk 1 above)
Comment (Wmg20) [reply CONTRIBUTERS.md]: okay, I will fix it

**Review by Wmg20 — COMMENTED** [review]
done

**Review by demonkillerr — APPROVED** [review]
Looks Good to me.

**Review by riyavishwa981 — APPROVED** [review]

**Pull request merged**